src/explore-education-statistics-common/src/modules/release/components/OfficialStatisticsSection.tsx

```diff
@@ -33,6 +33,25 @@ export default function OfficialStatisticsSection({
         </a>
         .
       </p>
+      <p>
+        Our statistical practice is regulated by the Office for Statistics
+        Regulation (OSR).
+      </p>
+      <p>
+        OSR sets the standards of trustworthiness, quality and value in the{' '}
+        <a href="https://code.statisticsauthority.gov.uk/the-code/">
+          Code of Practice for Statistics
+        </a>{' '}
+        that all producers of official statistics should adhere to.
+      </p>
+      <p>
+        You are welcome to contact us directly with any comments about how we
+        meet these standards. Alternatively, you can contact OSR by emailing{' '}
+        <a href="mailto:regulation@statistics.gov.uk">
+          regulation@statistics.gov.uk
+        </a>{' '}
+        or via the OSR website.
```

Comment:
I expected "OSR website" to be an href to the OSR website, but happy to leave it if this copy has been approved already.

Reply:
It's not an href in the JIRA ticket body, which is what I was working off. I agree it sounds like an improvement though - do we need to seek approval from Cam/Laura to deviate from the ticket or is this safe to add?

Reply:
Yeah I would check with them if you're interested in bringing it up.  Happy for this to go in either way of course, but they might appreciate the heads up.

Reply:
Would probably good for their Google Analytics to see some nice outbound links being used!

Reply:
Having discussed with Cam and Laura, this has now been added:
<img width="856" alt="with-osr-website-href" src="https://github.com/dfe-analytical-services/explore-education-statistics/assets/63285990/1b31f034-507f-49f1-9602-33fdb4ab0f78">